core/nodepool/config/templates/stack-template.json

```diff
@@ -349,7 +349,7 @@
                   "Resource": "arn:{{.Region.Partition}}:s3:::{{ .KubeResourcesAutosave.S3Path }}/*"
                 },
                 {{end}}
-                {{if .Kube2IamSupport.Enabled }}
+                {{if or .Kube2IamSupport.Enabled .KIAMSupport.Enabled }}
```

Comment:
@mumoshu sorry, this bit was incorrect - it's not needed (rest is fine though). Only the kiam server needs this. I'm going to remove it in https://github.com/kubernetes-incubator/kube-aws/pull/1285 if that's ok as there are some merge conflicts there.